main.go

```diff
@@ -90,7 +95,8 @@ func main() {
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "osd-metrics-exporter-lock",
-		Namespace:              "openshift-osd-metrics",
+		Namespace:              "",
+		NewCache:               cache.MultiNamespacedCacheBuilder(watchNamespaces),
```

Comment:
I think we can get rid of the `Namespace` line here due to the MultiNamespacedCacheBuilder.

https://sdk.operatorframework.io/docs/building-operators/golang/operator-scope/#watching-resources-in-a-set-of-namespaces